move_base/include/move_base/move_base.h

```diff
@@ -105,7 +105,7 @@ namespace move_base {
     private:
       /**
        * @brief  A service call that clears the costmaps of obstacles
-       * @param req The service request 
+       * @param req The service request
```

Comment:
It is a personal pet-peeve of mine when code has extraneous white space, and I'm aware that the navigation code violates this in all sorts of ways. However, for ease of review, its nicer when there are either only functional changes, or only formatting changes.